src/app/containers/App/App.jsx

```diff
@@ -99,14 +105,17 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
           pageType: nextPageType,
           loading: false,
           pageData: path(['pageData'], data),
+          remoteConfig: path(['remoteConfig'], data),
```

Comment:
Are we just going to pass remoteConfig in at the top level, instead of relying on this being returned by `getInitialData`? Otherwise I'm not sure there's much point doing this outside of getInitialData